fleet/view/js/detail_fleet.js

```diff
@@ -0,0 +1,1014 @@
+var address_bp, contact_person;
+var loc = window.location.pathname;
+var split_url = loc.split('/');
+var url_host = window.location.origin + "/" + split_url[1] + "/" + split_url[2];
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
+console.log(split_host[2]);
+var index = window.location.origin + "/" + split_url[1]
+
+console.log(url_host);
+
+var id = 0;
+var list_truck_type = [];
+var list_sub_truck_type = []; 
+var driver_arr = [];
+var driver = [];
+var list_tracker = [];
+var seq_doc;
+
+let doc_list_list, contract_list_list, food, field_man, maintenance_list_list, checklist_unit_list_list;
+
+if (split_host[2] != 'localhost') {
+    if (window.location.pathname.split('/')[3]) {
+        id = window.location.pathname.split('/')[3];
+    }
+} else {
+    if (window.location.pathname.split('/')[4]) {
+        id = window.location.pathname.split('/')[4];
+    }
+}
+
+$(document).ready(function () {
+    view();
+    documentFleet();
+    contract();
+    maintenance();
+    //checklist_unit();
+    $('#procurement').datepicker({
+        language: 'en',
+        dateFormat: 'yyyy'
+    });
+});
+
+function change_driver(cond) {
+    if (cond == "left") {
+        var id_sel_right = document.getElementById('right_driver').value;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400071)
```suggestion
        var id_sel_right = document.getElementById("right_driver").value;
```